Description: Relax missing_docs lint
 missing_docs lint has changed in recent rustc, and now reports some new
 entries as missing docs.
 .
 Downgrade the failure to a simple warning, to let the crate build anyway.
Forwarded: not-needed
Author: Luca Bruno <lucab@debian.org>
Last-Update: 2015-11-14

--- a/deps/aho-corasick-0.2.1/src/lib.rs
+++ b/deps/aho-corasick-0.2.1/src/lib.rs
@@ -118,7 +118,7 @@
 ```
 */
 
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 
 extern crate memchr;
 #[cfg(test)] extern crate quickcheck;
--- a/deps/docopt-0.6.67/src/lib.rs
+++ b/deps/docopt-0.6.67/src/lib.rs
@@ -213,7 +213,7 @@
 #![crate_name = "docopt"]
 #![doc(html_root_url = "http://burntsushi.net/rustdoc/docopt")]
 
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 
 extern crate regex;
 extern crate rustc_serialize;
--- a/deps/flate2-0.2.7/src/lib.rs
+++ b/deps/flate2-0.2.7/src/lib.rs
@@ -25,7 +25,7 @@
 //! already existing stream to chain construction.
 
 #![doc(html_root_url = "http://alexcrichton.com/flate2-rs")]
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 #![allow(trivial_numeric_casts)]
 #![cfg_attr(test, deny(warnings))]
 
--- a/deps/git2-0.2.12/src/lib.rs
+++ b/deps/git2-0.2.12/src/lib.rs
@@ -65,7 +65,7 @@
 
 #![doc(html_root_url = "http://alexcrichton.com/git2-rs")]
 #![allow(trivial_numeric_casts, trivial_casts)]
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 #![cfg_attr(test, deny(warnings))]
 #![cfg_attr(all(feature = "unstable", test, unix), feature(fs, fs_ext))]
 #![cfg_attr(feature = "unstable", feature(catch_panic))]
--- a/deps/memchr-0.1.3/src/lib.rs
+++ b/deps/memchr-0.1.3/src/lib.rs
@@ -3,7 +3,7 @@
 to the corresponding function in `libc`.
 */
 
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 
 extern crate libc;
 
--- a/deps/num_cpus-0.2.6/src/lib.rs
+++ b/deps/num_cpus-0.2.6/src/lib.rs
@@ -1,6 +1,6 @@
 //! Replaces the deprecated functionality of std::os::num_cpus.
 #![cfg_attr(test, deny(warnings))]
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 
 extern crate libc;
 
--- a/deps/regex-0.1.38/src/lib.rs
+++ b/deps/regex-0.1.38/src/lib.rs
@@ -389,7 +389,7 @@
 //! some other regular expression engines. (We pay for this by disallowing
 //! features like arbitrary look-ahead and back-references.)
 
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 #![cfg_attr(test, deny(warnings))]
 #![cfg_attr(feature = "pattern", feature(pattern))]
 #![doc(html_logo_url = "http://www.rust-lang.org/logos/rust-logo-128x128-blk-v2.png",
--- a/deps/regex-syntax-0.1.2/src/lib.rs
+++ b/deps/regex-syntax-0.1.2/src/lib.rs
@@ -60,7 +60,7 @@
 ```
 */
 
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 
 #[cfg(test)] extern crate quickcheck;
 #[cfg(test)] extern crate rand;
--- a/deps/semver-0.1.19/src/lib.rs
+++ b/deps/semver-0.1.19/src/lib.rs
@@ -121,7 +121,7 @@
 
 #![doc(html_logo_url = "http://www.rust-lang.org/logos/rust-logo-128x128-blk-v2.png",
        html_favicon_url = "http://www.rust-lang.org/favicon.ico")]
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 #![cfg_attr(test, deny(warnings))]
 
 // We take the common approach of keeping our own module system private, and
--- a/deps/tar-0.2.14/src/lib.rs
+++ b/deps/tar-0.2.14/src/lib.rs
@@ -8,7 +8,7 @@
 //! [1]: http://en.wikipedia.org/wiki/Tar_%28computing%29
 
 #![doc(html_root_url = "http://alexcrichton.com/tar-rs")]
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 #![cfg_attr(test, deny(warnings))]
 
 extern crate libc;
--- a/deps/term-0.2.9/src/lib.rs
+++ b/deps/term-0.2.9/src/lib.rs
@@ -60,7 +60,7 @@
        html_favicon_url = "http://www.rust-lang.org/favicon.ico",
        html_root_url = "http://doc.rust-lang.org/nightly/",
        html_playground_url = "http://play.rust-lang.org/")]
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 #![cfg_attr(test, deny(warnings))]
 #![cfg_attr(rust_build, feature(staged_api))]
 #![cfg_attr(rust_build, staged_api)]
--- a/deps/toml-0.1.21/src/lib.rs
+++ b/deps/toml-0.1.21/src/lib.rs
@@ -36,7 +36,7 @@
 //! [2]: https://github.com/BurntSushi/toml-test
 
 #![doc(html_root_url = "http://alexcrichton.com/toml-rs")]
-#![deny(missing_docs)]
+#![warn(missing_docs)]
 #![cfg_attr(test, deny(warnings))]
 
 #[cfg(feature = "rustc-serialize")] extern crate rustc_serialize;
